src/Command/InstallCommands.php

```diff
@@ -23,8 +23,13 @@ public function install($opts = ['existing-config' => false])
             $this->standardInstallation();
         }
 
-        // Workaround for https://www.drupal.org/project/drupal/issues/3091285.
-        $result = $this->taskExec('chmod u+w sites/default')
+        $result = $this->taskExecStack()
+            // Ensure resources directories exists and are writable.
+            ->exec('mkdir -p sites/default/files/uploaded_resources')
+            ->exec('mkdir -p sites/default/files/resources')
+            ->exec('chmod -R 777 sites/default/files')
+            // Workaround for https://www.drupal.org/project/drupal/issues/3091285.
+            ->exec('chmod u+w sites/default')
             ->dir(Util::getProjectDocroot())
```

Comment:
@clayliddell aren't these two lines redundant?

Reply:
No, the second line just affects `sites/default` since it's non-recursive. The first line is recursive and targets all files and folders within `sites/default/files`.

Reply:
I know, I just was thinking they could be combined but I can see how it might cast too wide a net to just 777 the whole default folder. OK will merge after testing

Reply:
There's a few settings files in the `sites/default` directory which are read-only for non-owners by default, and I think it would probably be best to leave them that way:
```sh
$ ls -al
drwxr-xr-x  3 clayliddell clayliddell  4096 Feb 17 13:48 .
drwxr-xr-x 11 clayliddell clayliddell  4096 Feb 17 13:48 ..
-rw-r--r--  1 clayliddell clayliddell  7565 Feb 17 13:48 default.services.yml
-rw-r--r--  1 clayliddell clayliddell 32852 Feb 17 13:48 default.settings.php
drwxrwxrwx  7 clayliddell clayliddell  4096 Feb 17 13:50 files
-rw-r--r--  1 clayliddell clayliddell   682 Feb 17 13:46 settings.docker.php
-r--r--r--  1 clayliddell clayliddell   403 Feb 17 13:46 settings.php
```